### PR TITLE
Extra fields in add_digital_object and timeout support

### PR DESCRIPTION
**Pull request by sevein**
The idea was to include these commits in #32 but we forgot!

### CODE REVIEW

File: agentarchives/atom/client.py [hunk 1]
```diff
@@ -42,9 +44,10 @@ class AtomClient(object):
     This change is due to the fact that slugs are visible by users whereas IDs aren't.
     """
 
-    def __init__(self, url, key):
+    def __init__(self, url, key, timeout=None):
         self.key = key
         self.base_url = urljoin(url, 'api/')
+        self.timeout = timeout or DEFAULT_TIMEOUT
```
Comment (Hwesta) [inline agentarchives/atom/client.py]: With a timeout parameter of 0, this will use DEFAULT_TIMEOUT - is this intentional?

Comment (sevein) [inline agentarchives/atom/client.py]: You're right. Actually I think I'm going to change it so the func parameter defaults to `DEFAULT_TIMEOUT`. I think this is a sane default given what's said in the docs:

> `timeout` is not a time limit on the entire response download; rather, an exception is raised if the server has not issued a response for `timeout` seconds (more precisely, if no bytes have been received on the underlying socket for `timeout` seconds).

What do you think?

**Comment by sevein** [comment]
Please review again!

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**